### PR TITLE
matrix remove obsolete SUSE

### PR DESCRIPTION
**Pull request by bruno-at-bareos**
- **matrix.yml: remove end of life SUSE/Leap**
- **tests: postgresql increase start timeout to 15**

#### Please check

- [X] Short description and the purpose of this PR is present _above this paragraph_
- [X] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)
- [Boy Scout Rule](https://docs.bareos.org/DeveloperGuide/generaldevel.html#boy-scout-rule)

### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)
Make sure you check/merge the PR using `devtools/pr-tool` to have some simple automated checks run and a proper changelog record added.

##### General
- [x] Is the PR title usable as CHANGELOG entry?
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted
- [x] Required backport PRs have been created
- [x] Correct milestone is set
- [x] depend on #1835 

##### Source code quality
- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR

### CODE REVIEW

**Comment by arogge** [comment]
I want this! :)
I think we should also backport this into Bareos 23 and 22 (which should be trivial)

**Review by arogge — APPROVED** [review]

**Comment by bruno-at-bareos** [comment]
> I want this! :) I think we should also backport this into Bareos 23 and 22 (which should be trivial)

So it mean I just got a green light to update my CD-Buildimage branch to make ready of 156 :-)
I will prepare that.

File: systemtests/tests/py3plug-fd-mariabackup/testrunner-default [hunk 1]
```diff
@@ -35,6 +35,8 @@ start_test
 mysql_init
 mysql_server_start
 
+cp -v my.cnf ~/.my.cnf || :
+
```
Comment (sebsura) [inline systemtests/tests/py3plug-fd-mariabackup/testrunner-default:39]: We should not do this.  People running our tests do not expect us to potentially destroy their configurations.  Lets see if this is really needed.

**Review by sebsura — CHANGES_REQUESTED** [review]

**Review by sebsura — APPROVED** [review]
ALGTM

**Pull request merged**